vagga.yaml

```diff
@@ -0,0 +1,28 @@
+containers:
+  pip-freeze:
+    setup:
+      - !Alpine v3.4
+      - !Py3Install
+        - pip
+        - Django
+        - gunicorn
+        - whitenoise
+        - django-environ
+      - !Sh pip freeze > requirements.txt
+  django:
+    environ:
+      DATABASE_URL: sqlite:///db.sqlite3
```

Comment:
Currently, you don't really need this as it's a default. You'll need to specify it in your next PR exposing postgres support